src/Commands/InstallCommand.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Rapidez\Statamic\Commands;
+
+use Illuminate\Console\Command;
+
+class InstallCommand extends Command
+{
+    protected $signature = 'rapidez-statamic:install';
+
+    protected $description = 'Install Rapidez Statamic';
+
+    public function handle(): void
+    {
+        $this->call('vendor:publish', [
+            '--tag' => 'runway-config',
+        ]);
+
+        $this->call('vendor:publish', [
+            '--provider' => 'Statamic\Eloquent\ServiceProvider',
+            '--tag' => 'statamic-eloquent-config',
+        ]);
+
+        $this->confirm('Did you set the table_prefix in config/statamic/eloquent-driver.php to "statamic_"?');
```

Comment:
Could we do this for the user?

Reply:
We can definitely verify this for the user. At least checking if `config('statamic.eloquent-driver.table_prefix')` is not empty